typesense-java/out/production/resources/logback.xml

```diff
@@ -0,0 +1,13 @@
+<configuration>
```

Comment:
`out` directory should probably be added to `.gitignore`?

Reply:
Thought I'd gitignored the output log directory but if its changed then yeah helps a bit when committing.